### PR TITLE
Fixes issue 7523 and 7657 with refactor of tutorials

### PR DESCRIPTION
**Pull request by lousanna**
See #7893.

This adds the relevant commits (2.5.x -> 2.6.x, and tutorial video series) that became missing from previous PR.

### CODE REVIEW

File: documentation/manual/gettingStarted/Tutorials.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ Play is non-opinionated about database access, and integrates with many object r
...
-* [play-scala-intro](https://github.com/playframework/play-scala-intro): This template uses [PlaySlick](https://www.playframework.com/documentation/2.5.x/PlaySlick) as part of a single Play project.
-* [Computer Database with Play-Slick](https://github.com/playframework/play-slick/tree/master/samples/computer-database): This template uses [PlaySlick](https://www.playframework.com/documentation/2.5.x/PlaySlick).  You will need to clone the `play-slick` project from Github and type `project computer-database-sample` in SBT to get to the sample project.
+* [play-scala-intro](https://github.com/playframework/play-scala-intro): This template uses [PlaySlick](https://www.playframework.com/documentation/2.6.x/PlaySlick) as part of a single Play project.
+* [Computer Database with Play-Slick](https://github.com/playframework/play-slick/tree/master/samples/computer-database): This template uses [PlaySlick](https://www.playframework.com/documentation/2.6.x/PlaySlick).  You will need to clone the `play-slick` project from Github and type `project computer-database-sample` in SBT to get to the sample project.
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/Tutorials.md]: I'm pretty sure you can use `[[PlaySlick]]` here. No need to add a complete link.

File: documentation/manual/gettingStarted/Tutorials.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ This is an example template showing Play with [Anorm](https://github.com/playfra
 
 #### Ebean
 
-This is an example template that uses [Ebean](https://ebean-orm.github.io/) using Play's [Ebean integration](https://www.playframework.com/documentation/2.5.x/JavaEbean). It also uses [Play-Bootstrap](https://adrianhurt.github.io/play-bootstrap/) for easy template scaffolding.
+This is an example template that uses [Ebean](https://ebean-orm.github.io/) using Play's [Ebean integration](https://www.playframework.com/documentation/2.6.x/JavaEbean). It also uses [Play-Bootstrap](https://adrianhurt.github.io/play-bootstrap/) for easy template scaffolding.
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/Tutorials.md]: The same here: try [[JavaEbean]].

File: documentation/manual/gettingStarted/Tutorials.md [hunk 3]
```diff
@@ -101,7 +101,7 @@ There are two examples shown here, but there are other compile time DI framework
 
 #### Manual Compile Time Dependency Injection
 
-This is an example template showing how to use manual compile time dependency injection and manual routing with the [SIRD router](https://www.playframework.com/documentation/2.5.x/ScalaSirdRouter), useful for minimal REST APIs and people used to Spray style routing:
+This is an example template showing how to use manual compile time dependency injection and manual routing with the [SIRD router](https://www.playframework.com/documentation/2.6.x/ScalaSirdRouter), useful for minimal REST APIs and people used to Spray style routing:
```
Comment (marcospereira) [inline documentation/manual/gettingStarted/Tutorials.md]: Use `[[SIRD router|ScalaSirdRouter]]`.

**Comment by lousanna** [comment]
Thanks @marcospereira!!! There doesn't seem to be a JavaEbean doc however.

**Comment by gmethvin** [comment]
JavaEbean (and PlaySlick) should exist, but only after we compile the full documentation. See [this comment](https://github.com/playframework/playframework.com/issues/86#issuecomment-335993953) for an explanation of how we do that.

Unfortunately our validation task doesn't know that we later expect to merge the documentation with the play-slick and play-ebean provided documentation, so we can't reference the PlaySlick or PlayEbean pages here.

A temporary fix would be to use`%PLAY_VERSION%` instead of 2.5.x or 2.6.x in the links, like `https://www.playframework.com/documentation/%PLAY_VERSION%/PlaySlick`. This will use whatever the current documentation version is.

Long-term we might want to have a more robust solution that allows us to still validate the docs but use the double bracket syntax. Maybe we can just have a list of exceptions, since this only happens in a few places.

**Comment by lousanna** [comment]
Got it, @gmethvin. Thank you so much for the good information and feedback

**Pull request merged**